### DISCUSSION TITLE
thank you serge <3

### DISCUSSION THREAD

**Discussion by hadixcanimm** [seerge/g-helper#3088] [General]
I sold my Asus laptop today, Serge. I've been using it ever since you developed Ghalper, and it keeps getting better. I wish my new laptop was an Asus so I could continue using it. You've created the best computer control panel I've ever seen. It's so good that I wished I could buy an Asus again, but it's too expensive in my country, so I went for the HP Victus instead. I hope I can find an alternative to Ghalper for HP. Thanks again, and good luck with the developments.

**Comment by seerge** [comment]
@hadixcanimm thanks for a kind feedback :)
